src/app/qgsattributetabledialog.cpp

```diff
@@ -1221,6 +1233,19 @@ void QgsAttributeTableDialog::toggleDockMode( bool docked )
     updateTitle();
     mDialog->restoreGeometry( QgsSettings().value( QStringLiteral( "Windows/BetterAttributeTable/geometry" ) ).toByteArray() );
     mDialog->show();
+
+    // restore attribute table shortcuts in window mode
+    QgsDebugMsg( QStringLiteral( "Restore attribute table dialog shortcuts in window mode" ) );
+    // duplicated on Main Window
+    mActionCopySelectedRows->setShortcut( QApplication::translate( "QgsAttributeTableDialog", "Ctrl+C", Q_NULLPTR ) );
```

Comment:
I don't believe we usually translate shortcuts

Reply:
Code generated by QT Creator makes these shortcuts translatable. If we keep the same policy in our written code, we will enable translatable shortcuts in the future.

For example, in Portuguese and Spanish, the usual shortcut for saving is <kbd>Ctrl</kbd>+<kbd>G</kbd> (Guardar, Gravar).

But **I don't have a strong opinion on this**. It it causes noise, I'll be happy to remove.

Reply:
I'd say remove it. There's hundreds of shortcuts set in the code which aren't translated already, this would require a coordinated approach to address.

Reply:
Done